packages/styles/scss/patterns/listsection/_listsection.scss

```diff
@@ -0,0 +1,106 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@mixin listsection { 
+  .#{$prefix}--listsection {
+    color: $text-01;
+    background: $ui-background;
+    &__container {
+      padding-left: $layout-01;
```

Comment:
What I did was reduced container padding to 1 rem and added 1 rem padding at sm breakpoint which give 1rem padding for sm breakpoint.

@include carbon--make-container adds 2rem to padding left & right. 
@include carbon--make-row adds -1rem of margin-right and margin-left.
We can use this combination to get 1 rem padding left & right for sm breakpoint.

Although both works the same I would prefer using carbon--make-container because it's more sementic approach. I am updating the code.

Thanks James for pointing out.

<img width="441" alt="Screen Shot 2019-10-29 at 9 22 27 AM" src="https://user-images.githubusercontent.com/8072521/67770946-bcd41a00-fa2d-11e9-8a4a-2eb64107c582.png">

Reply:
@sannminwin So the mixin provides more value than just semantics within `SCSS`, the gutters change at different breakpoints, and it handles that for us while also aligning us with Carbon. So if Carbon's grid changes we don't have to do anything in our code to but update the package.